docs/setting-up.md

```diff
@@ -65,7 +65,6 @@ More information can be found at [this documentation](https://help.github.com/ar
       **Windows users** should use a **forward slash**(`/`) instead of the Windows default **backward slash**(`\`) while specifying the path.
 
 If you followed every step correctly, you should have successfully set up the development environment.
```

Comment:
This sentence can be moved to step 5. Remember to delete the extra blank line after removing the sentence from this step so to maintain the consistency of at most one blank line in a row.